scvi/models/log_likelihood.py

```diff
@@ -38,13 +40,21 @@ def compute_reconstruction_error(vae, posterior, **kwargs):
     log_lkl = 0
     for i_batch, tensors in enumerate(posterior):
         sample_batch, local_l_mean, local_l_var, batch_index, labels = tensors[:5]  # general fish case
-        reconst_loss, kl_divergence = vae(sample_batch, local_l_mean, local_l_var, batch_index=batch_index,
-                                          y=labels, **kwargs)
+
+        if hasattr(vae, 'reconstruction_loss_fish'):
```

Comment:
Let's delay merging until this is handled in a general way

Reply:
Or can we merge if we adapt VAEF to `compute_reconstruction_error`, and not the other way around like done here ?

Reply:
Yes that's general enough!

Reply:
Great, thanks !

So I just changed VAEF in order to adapt it to `compute_reconstruction_error` and make the latter general.

What do you think about these last changes ?